lib/plugins/aws/provider/awsProvider.js

```diff
@@ -194,6 +194,13 @@ class AwsProvider {
     impl.addEnvironmentProfile(result, `AWS_${stageUpper}`);
 
     result.region = this.getRegion();
+
+    const deploymentBucketObject = this.serverless.service.provider.deploymentBucketObject;
+    if (deploymentBucketObject && deploymentBucketObject.serverSideEncryption
```

Comment:
If you have your own encryption certs your are storing there may be a reason iirc...